code/datums/materials/basemats.dm

```diff
@@ -60,10 +62,11 @@
 	categories = list(MAT_CATEGORY_ORE = TRUE, MAT_CATEGORY_RIGID = TRUE)
 	sheet_type = /obj/item/stack/sheet/mineral/uranium
 	coin_type = /obj/item/coin/uranium
+	armor_modifiers = list("melee" = 2, "bullet" = 2, "laser" = 0.5, "energy" = 0.5, "bomb" = 0, "bio" = 0, "rad" = 0, "fire" = 1, "acid" = 1)
```

Comment:
these melee and bullet armor values seem a bit high, even with the radioactive downside to armors made out of this

Reply:
You'll die of radiation before you can do much.

Reply:
Unless you're a radiation proof species?

Reply:
Epic plasmeme gamers

Reply:
oh yeah fuck lmao lemme smash that nerf button